model/user-agent/registry.yaml

```diff
@@ -35,3 +35,83 @@ groups:
           using a user-agent for non-browser products, such as microservices with multiple names/versions inside the
           `user_agent.original`, the most significant version SHOULD be selected. In such a scenario it should align
           with `user_agent.name`
+
+  - id: registry.user_agent.os
+    type: attribute_group
+    display_name: User-agent OS Attributes
+    brief: "Describes the OS user-agent attributes."
+    attributes:
+      - id: user_agent.os.type
+        type:
+          members:
+            - id: windows
+              value: 'windows'
+              brief: "Microsoft Windows"
+              stability: experimental
+            - id: linux
+              value: 'linux'
+              brief: "Linux"
+              stability: experimental
+            - id: darwin
+              value: 'darwin'
+              brief: "Apple Darwin"
+              stability: experimental
+            - id: freebsd
+              value: 'freebsd'
+              brief: "FreeBSD"
+              stability: experimental
+            - id: netbsd
+              value: 'netbsd'
+              brief: "NetBSD"
+              stability: experimental
+            - id: openbsd
+              value: 'openbsd'
+              brief: "OpenBSD"
+              stability: experimental
+            - id: dragonflybsd
+              value: 'dragonflybsd'
+              brief: "DragonFly BSD"
+              stability: experimental
+            - id: hpux
+              value: 'hpux'
+              brief: "HP-UX (Hewlett Packard Unix)"
+              stability: experimental
+            - id: aix
+              value: 'aix'
+              brief: "AIX (Advanced Interactive eXecutive)"
+              stability: experimental
+            - id: solaris
+              value: 'solaris'
+              brief: "SunOS, Oracle Solaris"
+              stability: experimental
+            - id: z_os
+              value: 'z_os'
+              brief: "IBM z/OS"
+              stability: experimental
+        brief: >
+          The operating system type.
+        stability: experimental
+      - id: user_agent.os.description
```

Comment:
is this needed? I don't see it in ECS, and I'm not sure what you would populate it with via https://github.com/ua-parser/uap-core

Reply:
I think the equivalent in ECS would be the `os.full` field, ECS embeds all the `os` fields in the user_agent: https://www.elastic.co/guide/en/ecs/current/ecs-user_agent.html#_field_reuse_30

Given that there is no direct mapping for this field in https://github.com/ua-parser/uap-core/blob/master/docs/specification.md#os_parsers and the collector's use case does not cover this attribute at the moment, it is ok for me to remove it until the use case https://github.com/open-telemetry/semantic-conventions/pull/1434/commits/1b46946a1d752f6e7fc157c22ec6670262eb7b11